src/main/fc/rc_adjustments.h

```diff
@@ -100,6 +98,7 @@ PG_DECLARE_ARRAY(adjustmentRange_t, MAX_ADJUSTMENT_RANGE_COUNT, adjustmentRanges
 void resetAdjustmentStates(void);
 void updateAdjustmentStates(void);
 struct controlRateConfig_s;
-void processRcAdjustments(struct controlRateConfig_s *controlRateConfig);
+int processRcAdjustments(struct controlRateConfig_s *controlRateConfig);
```

Comment:
The return value of this function is not used, so it should remain `void`.

Reply:
It was a workaround to avoid the compiler warning about the "unused" newValue (which is only used through the precompiler condition). Any suggestion for another way to avoid it without code duplication?

Reply:
You could use:
```
#if defined(OSD) && defined(USE_OSD_ADJUSTMENTS)
     osdShowAdjustment(adjustmentLabels[adjustmentFunction], newValue);
#else
    UNUSED(newValue);
#endif
```

which also makes it explicit what is going on.

Reply:
Thx, done.